R/globals.R

```diff
@@ -0,0 +1,3 @@
+utils::globalVariables(c("Freq", "AtomicMass", "Freq.x",
```

Comment:
I don't think we should declare these as global variables. Some of these are column names used in dataframes.

Reply:
I was getting this warning which prevents ci-cd github action and so those columns names used in dataframes are added as the global variables. https://community.rstudio.com/t/how-to-solve-no-visible-binding-for-global-variable-note/28887